python_src/Dockerfile

```diff
@@ -5,22 +5,24 @@ ENV PYTHONDONTWRITEBYTECODE 1
 # Turns off buffering for easier container logging
 ENV PYTHONUNBUFFERED 1
 
-# Install dev dependencies
+# Install non python dependencies
 RUN apt-get update
 RUN apt-get install -y libpq-dev gcc curl
 
 # Install poetry
 RUN pip install -U pip
 RUN pip install "poetry==1.1.14"
 
-COPY . /performance_manager/
-WORKDIR /performance_manager/
-RUN poetry config virtualenvs.create false
```

Comment:
we want to create the virtual env now so that we can get `poetry run <pipeline_name>`to work correctly.